.buildkite/code.pipeline.yml

```
@@ -86,6 +86,9 @@ retry: &retry_agent_failure
     # Buildkite lost contact with the agent or it stopped reporting.
     - exit_status: -1
       limit: 2
+    # Cargo build error (e.g., Unable to update registry `crates-io`).
+    - exit_status: 101
+      limit: 10
```

```suggestion
      limit: 2
```
I would suggest a lower retry limit, because 10 retries multiplied by the number of parallel build jobs (without exponential back-off) will definitely not help an already-failing external service to recover.

---

I'm testing what happens if I set limit to 10. Previous version used 2, and all retries failed so it did not solve anything.